cmake/Install.cmake

```diff
@@ -6,7 +6,12 @@
 #
 #
 
-if(NOT INSTALLED_ENGINE)
+ly_set(LY_INSTALL_ENABLED TRUE)
+if(INSTALLED_ENGINE)
+    ly_set(LY_INSTALL_ENABLED FALSE)
+endif()
```

Comment:
Should we rename this option?
When the SDK build has support for building a Project Release Game Layout, I could see some of these functions needing to be called.

Perhaps the option should be `LY_INSTALL_ENGINE_ENABLED` to emphasize that it is used to gate the Engine Source install logic?

Reply:
this is actually enabling or not any install. Once we want to do a project install from an engine SDK we would have to enable this somehow.
Eventually this variable will go away once we support installing engine/project from source/prebuilt